### DISCUSSION TITLE
Recording other satellites with Raspberry NOAA V2

### DISCUSSION THREAD

**Discussion by Kamal-4S7AB** [jekhokie/raspberry-noaa-v2#412] [Q&A] [answered]
This is my first post here. I have installed Raspberry NOAA V2 in my RPi4 and works great with QFH and RTL_SDR V3.

I am thinking whether it is possible to record signals of other sats like CW beacons of RS44 or anything similar. Has someone done this before or can someone guide me on how to do it. Then I can give a try.

Recording wave files would be sufficient for the moment. 

Regards
Kamal / 4S7AB

**Comment by dom-robinson** [comment, marked as answer]
Hey Kamal - The fundamental RTLSDR software that the RN2 scripts drive can record any frequency that the SDR card can be tuned to. However i think you might do better to build up some scripts aroudn the RTLSDR drivers specifically since the RN2 setup is broadly all about the scheduling of 'when' to use the RTLSDR capture, avoiding scheduling conflicts and processing ('demodulating') the captured audio into images, filing and presenting them to the webpanel.

So yes, you potentially could adapt the scripts to drive that fundamental capture, but IMHO it would be like buying a car to use the headlights to read a book (!) 

I suggest you have a look at how to directly use the RTLSDR command line to capture a specific frequency to .wav

eg [https://raspberrypi.stackexchange.com/questions/14433/record-audio-from-rtl-sdr-using-usb-dongle](https://raspberrypi.stackexchange.com/questions/14433/record-audio-from-rtl-sdr-using-usb-dongle)

Once you have that running manually, if you still want to start to work that into the RN2 scheduling you would want to start in the ./scripts folder and work out from there...

Hope that helps...

**Reply by Kamal-4S7AB** [reply]
Hi Dom, 
Yes this helps. I looked at the links you mentioned too.
The way RN2 presents information is hard to stay away from. That's why I tried to do this. I can run the RTL_SDR script and listen to FM stations on my bluetooth speaker with Rpi4. That works well.

When I looked at the scripts, it is a bit hard for me to fully understand scripting. Also one script is inked to many others I think. If I get a good free time, I will give a try. 

Thanks for your ideas Dom.

Regards
Kamal
